lib/model/model.go

```diff
@@ -1242,65 +1241,55 @@ func (m *Model) ConnectedTo(deviceID protocol.DeviceID) bool {
 }
 
 func (m *Model) GetIgnores(folder string) ([]string, []string, error) {
-	var lines []string
-
 	m.fmut.RLock()
 	cfg, ok := m.folderCfgs[folder]
 	m.fmut.RUnlock()
 	if !ok {
-		return lines, nil, fmt.Errorf("Folder %s does not exist", folder)
-	}
-
-	if !cfg.HasMarker() {
-		return lines, nil, fmt.Errorf("Folder %s stopped", folder)
-	}
-
-	fd, err := os.Open(filepath.Join(cfg.Path(), ".stignore"))
-	if err != nil {
-		if os.IsNotExist(err) {
-			return lines, nil, nil
+		cfg, ok = m.cfg.Folders()[folder]
+		if ok {
```

Comment:
There is a difference between the folder config being in the model or the config: In case it is in the model the ignores can be directly loaded from memory. In case iti is only in the config (i.e. paused), it has to be read from ".stignore".